### DISCUSSION TITLE
如何更新版本热部署

### DISCUSSION THREAD

**Discussion by 2294648043** [apache/apisix#7093] [Q&A]
正在技术选型阶段，目前最后一个问题是如果更新APISIX版本，APISIX本身进行版本升级，例如从1.x升级到2.x，没有做集群，怎么可以实现不影响线上业务的情况做热更新。这是一个问题吗？

**Comment by tzssangglass** [comment]
1. Upgrading from 1.x to 2.x is an incompatible upgrade, and as far as I know, many people choose to stop upgrading for insurance purposes.
2. The capabilities required for version hot deployment are available from APISIX, but the implementation options are numerous, and it is up to the user to decide.
